packages/x-charts/src/ChartsLabel/ChartsLabelMark.tsx

```diff
@@ -6,12 +6,18 @@ import clsx from 'clsx';
 import { ChartsLabelMarkClasses, labelMarkClasses, useUtilityClasses } from './labelMarkClasses';
 import { consumeThemeProps } from '../internals/consumeThemeProps';
 
+export interface ChartsLabelCustomMarkProps {
+  className?: string;
+  /** Color of the series this mark refers to. */
+  color?: string;
+}
```

Comment:
I wonder if we should pass the `seriesId` here. Otherwise if someone is using the same component for all series they won't know which series is rendering the current component.

Reply:
Could be an option yeah

Reply:
@alexfauquette what's your take? 

If the decision is affirmative, I'll do that in a follow-up PR so I can merge this PR faster.

Reply:
I don't see a reason to modify the component based on the id. The series type already make more sense.

I fear that with passing ids, people start do do complex stuff relying on ids, and then realise small id change can break their entire legend design